engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/JobManager.java

```diff
@@ -132,6 +153,7 @@ protected void hintJobExecutor(JobEntity job) {
       job.setLockOwner(jobExecutor.getLockOwner());
       transactionListener = new ExclusiveJobAddedNotification(job.getId(), jobExecutorContext);
     } else {
+      // schedule job for the next processor and reset Acquisition strategy
```

Comment:
`schedule job for the next processor`: what exactly do you mean here?

Reply:
I was trying to use the same terminology as the one on `line 147` (`lock job & add to the queue of the current processor`) so that it's clear in the future that the `else` block, with the `MessageAddedNotification` will not result in the immediate execution of a Job, but will reset the (Backoff) acquisition strategy to 0 wait time. This will result in the above job being executed on the next run.

Reply:
@ThorbenLindhauer please check the comment now.